src/main/webapp/javascript/tests.js

```diff
@@ -0,0 +1,145 @@
+/**
+ * TODO(matwsuaz): update with a new method to return refereces to live HTML DOM
+ * elements to reduce redundancy and code length.
+ * TODO(matwsuaz): update HTMl injection test to provide test result description
+ * and inform which test failed specifically.
+ */
+runTests = () => {
+    testCommentCard();
+}
+
+/**
+ * Makes a call to create a recipe card and makes sure the result is what is
+ * expected.
+ */
+testCommentCard = () => {
+    // The div in the test file for rendering the comment card test results
+    const testDiv = "recipe-card-test";
+
```

Comment:
nit: indentation should be +2 everywhere
[JavaScript Style Guide](https://google.github.io/styleguide/jsguide.html#formatting-block-indentation)

Reply:
Ack and fixed